### PR TITLE
New releases (all betas, with client builder support):

### PR DESCRIPTION
**Pull request by jskeet**
- BigQuery
- BigQuery Data Transfer
- Bigtable (including admin and common packages)
- Grafeas and ContainerAnalysis
- Dialogflow
- KMS
- Language
- Storage
- Tasks
- Translation
- Video Intelligence
- Vision

### CODE REVIEW

File: apis/apis.json [hunk 1]
```diff
@@ -809,7 +809,7 @@
     "id": "Google.Cloud.Tasks.V2",
     "productName": "Google Cloud Tasks",
     "productUrl": "https://cloud.google.com/tasks/",
-    "version": "1.1.0-beta00",
+    "version": "1.1.0-beta01",
```
Comment (chrisdunelm) [inline apis/apis.json]: Just to check that this is definitely intended to be released?

File: apis/apis.json [hunk 2]
```diff
@@ -422,7 +422,7 @@
     "id": "Google.Cloud.Kms.V1",
     "productName": "Google Cloud Key Management Service",
     "productUrl": "https://cloud.google.com/kms/",
-    "version": "1.1.0-beta00",
+    "version": "1.1.0-beta01",
```
Comment (chrisdunelm) [inline apis/apis.json]: Just to check that this is definitely intended to be released?

**Comment by jskeet** [comment]
Yes to both of the "definitely release". They were at beta00 as we knew we had new features.

**Comment by chrisdunelm** [comment]
Great, then definitely LGTM.

**Pull request merged**